src/selectors/entities/typing.js

```diff
@@ -11,25 +11,38 @@ import {getTeammateNameDisplaySetting} from 'selectors/entities/preferences';
 
 import {displayUsername} from 'utils/user_utils';
 
+const getUsersTypingImpl = (profiles, teammateNameDisplay, channelId, parentPostId, typing) => {
```

Comment:
`getUsersTypingImpl` does not sound obvious to me what it means. 0/5 on what name it should be.

Reply:
Yes, I haven't a better proposal, and I understood that was clear enough from the context.